sema/harvest/__main__.py

```
@@ -169,13 +156,13 @@
     store.insert(graph, "urn:harvest:context")
 
 
-def make_service(args) -> service:
+def make_service(args) -> HarvestService:
```

Ruff

Missing type annotation for function argument `args`

(ANN001)